book/developers/common-issues.md

```diff
@@ -12,7 +12,7 @@ This is likely due to two different versions of `alloy_sol_types` being used. To
 
 ```toml
 [dependencies]
-sp1-sdk = { version = "1.0.1", default-features = false }
```

Comment:
can we maybe make these placeholder so we don't need to update every version?

Reply:
hmm I will look into this later